src/main/java/uk/gov/hmcts/reform/cwrdapi/util/MappingField.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.cwrdapi.util;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.RUNTIME)
+public @interface MappingField {
+
+    String columnName() default "";
+
+    Class clazz() default Object.class;
+
+    int noOfCollectionProperties() default 0;
```

Comment:
Looks like this is not used anywhere.

Reply:
Yes is domain and we are planning to use for spitting in sync with 1952.

Reply:
cool then :)